packages/client/src/l2.write.ts

```diff
@@ -21,27 +22,19 @@ import { abi as l2Abi } from '@blockful/contracts/out/L2Resolver.sol/L2Resolver.
 import { abi as uAbi } from '@blockful/contracts/out/UniversalResolver.sol/UniversalResolver.json'
 import { getRevertErrorData, handleL2Storage, getChain } from './client'
 
-const program = new Command()
-program
-  .option('-r --resolver <address>', 'ENS Universal Resolver address')
-  .option('-p --provider <url>', 'web3 provider URL', 'http://127.0.0.1:8545/')
-  .option(
-    '-pl2 --providerL2 <url>',
-    'web3 provider URL for layer2',
-    'http://127.0.0.1:8547',
-  )
-  .option('-i --chainId <chainId>', 'chainId', '1337')
-  .option(
-    '-pk --privateKey <privateKey>',
-    'privateKey',
-    '0xb6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659', // local arbitrum PK
-  )
-  .option('-l2r --l2resolver <l2resolver>', 'l2resolver')
-
-program.parse(process.argv)
-
-const { provider, providerL2, chainId, privateKey, l2resolver } = program.opts()
-let { resolver } = program.opts()
+config({
+  path: process.env.ENV_FILE || '../.env',
+})
+
+let {
+  UNIVERSAL_RESOLVER: resolver,
+  CHAIN_ID: chainId = '31337',
+  RPC_URL: provider = 'http://127.0.0.1:8545/',
```

Comment:
If the RPC_URL is different in the .env file, how is it getting the variable?

Reply:
the `'http://127.0.0.1:8545/'` is the default value, which means it is only set when the initial value is unset